app/src/app.scss

```diff
@@ -35,6 +35,15 @@ $theme-colors: map-merge($theme-colors, $custom-colors);
     --button-hover-border: #e3daf8;
 }
 
+.finalTextBox textarea {
+    border: 0;
+    outline: 0;
+    padding: 5px;
+    textarea:focus {
+        outline: none;
+    }
+}
+
```

Comment:
For speed I'm going to let this merge, but it _should_ have been incorporated into the `StyledComponent` class we already have.